atomic_reactor/plugins/exit_delete_from_registry.py

```diff
@@ -173,6 +185,10 @@ def run(self):
             orchestrator_delete = self.handle_worker_digests(session, worker_digests,
                                                              deleted_digests)
 
+            # Remove manifest from the registry
+            if orchestrator_delete:
+                self.delete_manifest_lists(session, registry_noschema, deleted_digests)
```

Comment:
Is it possible to arrange for the manifest lists to be deleted *before* the image manifests? Deleting them after leaves a small window of opportunity in which requests for that manifest list can fail (for example, if the client does not indicate they accept manifest lists).

Reply:
Right, that's a good idea. Perhaps this should happen in `handle_worker_digests`